main.py

```diff
@@ -131,6 +130,6 @@ def play_game(self):
             self.clean_up()
 
 
-if __name__ == 'main':
+if __name__ == '__main__':
```

Comment:
Looks like I told you how to do this wrong. My bad. Lol.

Reply:
All good. Twas but a short google away. lol